core/src/main/java/org/apache/iceberg/TableMetadata.java

```diff
@@ -348,6 +375,22 @@ public int defaultSpecId() {
     return defaultSpecId;
   }
 
+  public int defaultSortOrderId() {
+    return defaultSortOrderId;
+  }
```

Comment:
Should we also expose `defaultSpecId` directly? Normally, we use `spec().specId()`.

Reply:
I am following what we have for specs to be consisent.